packages/docusaurus-theme-classic/src/theme/BlogPostItem/index.js

```diff
@@ -45,7 +51,7 @@ function BlogPostItem(props) {
     return (
       <header>
         <h1 className={classnames('margin-bottom--sm', styles.blogPostTitle)}>
-          <Link to={permalink}>{title}</Link>
+          {isBlogPostPage ? title : <Link to={permalink}>{title}</Link>}
```

Comment:
Is there a better solution? Comparison with `location.pathname` will not work, what other options are there?

Reply:
I think this is the right solution (?). Determining whether it's a list or not using pathname parsing can lead to subtle bugs

Reply:
I agree this is a right way to check.